mmv1/third_party/terraform/tests/resource_container_node_pool_test.go.erb

```diff
@@ -2160,6 +2160,9 @@ resource "google_container_node_pool" "with_auto_pod_cidr" {
 	create_pod_range    = true
     pod_range           = "auto-pod-range"
 	pod_ipv4_cidr_block = "10.2.0.0/20"
+	node_performance_config = {
```

Comment:
I think this should be `network_performance_config`?

Reply:
Yes, fixed